content/docs/17-glossary-all.md

```diff
@@ -83,6 +92,10 @@ PaletteOS is a real-time operating system provisioned by Palette. It is embedded
 ## Palette Upgrade Controller
 
 A Kubernetes controller to be installed into the workload cluster to facilitate upgrades to new P6OS image.
+
+## Palette Virtual Cluster
+Palette Virtual Clusters enable operations teams to partition a host Kubernetes cluster and deploy lightweight virtual clusters on top, similar to how virtualization creates logically isolated virtual servers on top of physical machines. This is great for giving developers quick access to a sandbox environment for testing their code. Virtual clusters provide as strong a level of separation without introducing complicated overhead, such as separating physical resources and managing namespaces with complex RBAC configurations. Palette Virtual Clusters is powered by [vCluster](https://www.vcluster.com/). 

```

Comment:
If we can give a link to host clusters (host cluster definition within the glossary) here would be great.

Reply:
@resmaa can you provide more context? How would you integrate host clusters into the definition?

Reply:
A host cluster is just a regular kubernetes cluster that is used to host virtual clusters.